core/quantity/src/test/Formatting.test.ts

```diff
@@ -1223,3 +1235,190 @@ describe("Formatting tests:", () => {
     });
   });
 });
+
+describe("Formatting tests with more extensive units:", () => {
```

Comment:
This is my only nitpick, we should put a better test name. Maybe "Flips sign of value during formatting". Extensive isn't a good description of what this tests.